UserService/Controllers/UserController.cs

```diff
@@ -71,4 +72,28 @@ public IActionResult GetUser([FromRoute]string id)
             return StatusCode(500, $"An unexpected error occurred: {ex.Message}");
         }
     }
+
+    [HttpPut("{id}")]
+    public IActionResult UpdateUser([FromRoute] string id, [FromBody]UserDtoUpdate userUpdateDto)
+    {
+        if(!ModelState.IsValid) return BadRequest();
+
+        try
+        {
+            if(!_userRepository.UserExists(id)) return NotFound();
+
+            var user = _mapper.Map<User>(userUpdateDto);
+
+            user.Id = id;
+
+            _userRepository.UpdateUser(user);
```

Comment:
maybe use the boolean returned by this method

Reply:
We'll use it for wich purpose ?
It is more common in controller endpoints to return NoContent for PUT methods.